### DISCUSSION TITLE
Bug Hunter Achivement

### DISCUSSION THREAD

**Discussion by JhonnyLusonode** [community/community#19659] [Other Feature Feedback, Questions, & Ideas]
Hello everyone, 
I belive it would be a great idea putting some more achivements for example bug hunter when you report a bug on github or something related to that.
That would make the community more active trying to find some bugs/vulnerabilities on github which leads to a more secure environment.

What are your thoughts on this?

**Comment by matigodoy** [comment]
i think its a good idea, it promotes coding collaboration between developers!

**Reply by JhonnyLusonode** [reply]
Yea i think so too!
Thank you for your opinion!

**Comment by D7EAD** [comment]
Hi!

There already exists a Bug Hunter highlight for people who find bugs on any of Github's allowed penetration testing scopes; you can learn about that particular highlight [here](https://savelev.medium.com/improving-github-profile-with-highlights-and-achievements-be5fcdd558d). While it may not be a badge, it is certainly a rare item to see on someone's profile!

**Reply by JhonnyLusonode** [reply]
Hello @D7EAD , thank you for bringing this up!
i´ve seen it before but it´s not really an achivements but a highlight. 
Usually those highlights are attributed to those who reports a vulnerability and meets the requirements of Severity Guidelines.
The achivements i´m refering to it´s those visual bugs for example a bad redirect to a url, button malfunction, nav bar bugged etc..
And it would be a great idea putting like a badge with levels. The more you find your badge level ups.
Some of you might think this is a waste of time or it´s to childish but i believe it would be a nice contribution to the github community.

**Reply by melvincwng** [reply]
@JhonnyLusonode, cool idea +1 👍🏻

**Reply by JhonnyLusonode** [reply]
@melvincwng
Thank you for your opinion! 👍

**Comment by HauXun** [comment]
Contact 'Github Support' if you want this idea to come to fruition.

The community is free to participate and provide their own observations on how new achievements are announced, in case I may have missed something.

Hope this helped you ❤!

**Reply by JhonnyLusonode** [reply]
Hi @HauXun 
i´ll try doing it.
Thanks!

**Comment by HauXun** [comment]
As my previous answer you will have to suggest your idea with 'Github Support' or another method.
And yes, it's definitely not easy to do this in just a group chat like this, but it can be done if you submit suggestions, I send suggestions and lots of other people send suggestions to Github if they agree with your idea. And of course Github will consider this idea and implement it.

The idea is pretty cool, like Apple holding contests to test the security of their systems 😍

**Reply by JhonnyLusonode** [reply]
I´ll suggest my idea to the github support.
Thank you for your opinion!

**Comment by fajarhide** [comment]
maybe from github can make a badge for the achievement of Bug Hunter

badge reference : https://bugbounty.sa/
![logo-colored-bugbounty a9cce361](https://user-images.githubusercontent.com/4769529/176504236-604d5663-a3a0-49d8-81af-0f5501560a23.svg)

**Reply by JhonnyLusonode** [reply]
Yes that´s a good example!
Thank you for your contribution! 👍

**Comment by sinkaroid** [comment]
Hard to say but seems they will not consider this, The [Security Bug Bounty Hunter](https://bounty.github.com/) already take it all in the Highlights section instead badges, and the official orgs [GitHub Security Bug Bounty](https://github.com/GitHubBounty) are exists.  

And for security environment stuff, The [GitHub Advisory](https://github.com/advisories) also exists for advisoring vulnerability arround dependencies in each technologies, It's turn out the [Security advisory credit](https://github.com/advisories?query=nodejs) also takes in the Highlights section  

Both of them were already taken in the Highlights. The Github staff probably will not consider this idea, because it's reinventing the wheel.
![Screenshot_591](https://user-images.githubusercontent.com/12372481/176979525-af0f5529-6c1c-4368-9f70-55e9024f75e5.png)

**Comment by DumbergerL** [comment]
Great Idea!
